FLEDGE.md

```diff
@@ -151,28 +150,35 @@ const myAuctionConfig = {
                         'https://www.another-buyer.com': 200,
                         '*': 150,
                         ...},
+  'componentAuctions': [
+    {'seller': 'https://www.some-other-ssp.com',
+      ...},
+    ...
+  ]
 };
 const auctionResultPromise = navigator.runAdAuction(myAuctionConfig);
 ```
 
 
 This will cause the browser to execute the appropriate bidding and auction logic inside a collection of dedicated worklets associated with the buyer and seller domains.  The `auctionSignals`, `sellerSignals`, and `perBuyerSignals` values will be passed as arguments to the appropriate functions that run inside those worklets — the `auctionSignals` are made available to everyone, while the other signals are given only to one party.
 
+In some cases, multiple SSPs may want to run their own auction and send the winner up to another auction. To facilitate these "component auctions", `componentAuctions` can optionally contain additional auction configurations for each seller's "component auction".  The winning bid of each of these "component auctions" will be passed to the top-most auction.  How bids are scored in this case is further described in [2.4 Scoring Bids in Component Auctions](#24-scoring-bids-in-component-auctions). The `AuctionConfig` of component auctions may not have their own `componentAuctions`.
+
 The returned `auctionResultPromise` object is _opaque_: it is not possible for any code on the publisher page to inspect the winning ad or otherwise learn about its contents, but it can be passed to a Fenced Frame for rendering.  (The [Fenced Frame Opaque Source explainer](https://github.com/shivanigithub/fenced-frame/blob/master/OpaqueSrc.md) has initial thoughts about how this could be implemented.)  If the auction produces no winning ad, the return value can also be null, although this non-opaque return value leaks one bit of information to the surrounding page.  In this case, for example, the seller might choose to render a contextually-targeted ad.
 
 Optionally, `perBuyerTimeouts` can be specified to restrict the runtime (in milliseconds) of particular buyer's bidding scripts. If no value is specified for a particular buyer, a default timeout of 50 ms will be selected. Any `perBuyerTimeouts` higher than 500 ms will be clamped to 500 ms. A key of `'*'` is used to change the default of unspecified buyers.
 
 A `Permissions-Policy` directive named "run-ad-auction" controls access to the `navigator.runAdAuction()` API.
 
+In the case of a component auction, all `AuctionConfig` parameters for that component auction are only scoped to buyer and seller scripts run as part of that auction component. Similarly, all vaues specified by the top-level auction are not applied to the component auctions.
```

Comment:
What happens if the top-level auction configures both direct buyers as well as a number of component auctions? Would that mean that in the top-level auction, individual bids of such direct buyers compete against winning bids from each of the component auctions? It may be worthwhile to clarify the behavior.

Reply:
Yes, that's exactly the intention.  I've added a mention of this in section 2.4, further down.

Reply:
Marking this as resolved, though if you think this could be clearer, suggestions welcome!